torch/csrc/autograd/function.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <c10/util/ThreadLocal.h>
 #include <torch/csrc/autograd/engine.h>
+#include <torch/csrc/autograd/graph_task.h>
```

Comment:
Defines these methods in cpp to avoid `graph_task.h` get included in `function.h`, which is included in many more headers.